lib/api/Healthcheck.js

```diff
@@ -84,6 +94,14 @@ class Healthcheck {
                 topics[this._repConfig.topic] = topicMd;
                 connections.isrHealth = this._checkISRHealth(topicMd);
             }
+            if (topicMd && this._isMissingISR(topicMd)) {
+                const error = {
+                    method: 'Healthcheck.getHealthcheck',
+                    error: 'partition missing in-sync replica',
```

Comment:
Sorry to come back again but I still see this message as ambiguous, my first understanding of this message is "ther e is a missing in-sync replica to a partition" while I should read "there is no in-sync replica for a partition", correct?

Reply:
Okay, I think the confusion is around whether the wording communicates a count of in-sync replicas. Using 'no in-sync replica' will avoid that problem.